index.html

```diff
@@ -0,0 +1,555 @@
+                       <li><a href="#">Fundamental</a></li>
+                       <li><a href="#">ReadMe</a></li>
+                       <li><a href="#">Growth Quarters</a></li>
+                       <li><a href="#">Neural</a></li>
+                    </ul>
+                    <div class="border-icon">
+                        <div class="icon-border1"><i class="search icon"></i></div>
+                        <div class="icon-border2"><i class="bars icon"></i></div>
+                    </div>
+                      
+               </div>
+               <p class="p1"><a href="#">Did you know we have a TECH FESTIVAL? Check it out <i class="long arrow alternate right icon"></i></a></p>
+           </div>
+            
+            <div class="main-grid">
+                <div class="tnw">
+                    <div>
+                        <img src="./images/tnw-w.png" alt="tnw-logo"/>
+                        <time datetime="2020-03-13 20:00">Friday - March 13, 2020</time>
+                    </div> 
+                    <article>
+                        <h3>TECH</h3>
+                        <h2>How to handle toxic people as an<br> online community manager</h2>
+                        <div class="time-author">
+                          <p>by Milena Abrosimova</p>
+                          <p><i class="clock outline icon"></i>50 minutes ago</p>
+                          <p><i class="share alternate icon"></i>0</p>
+                        </div>
+                    </article>
+                </div>
+                <div class="reddit">
+                  <article>
+                      <h2>Reddit, Paypal, Spotify and more <br> are speaking at TNW's tech festival in Amsterdam</h2>
+                  </article>  
+                </div>
+                <div class="microsoft">
+                    <article>
+                        <h2>Satoshi Nakaboto: 'Bitcoin drops <br> a jaw-dropping 37% in 24 hours'</h2>
+                        <div class="time-author">
+                          <p>by Satoshi Nakaboto</p>
+                          <p><i class="clock outline icon"></i>2 hrs ago</p>
+                          <p><i class="share alternate icon"></i>5</p>
+                        </div>
+                    </article>
+                </div>
+
+                <section>
+                    <h2>Latest New <i class="chevron right icon"></i></h2>
+                    <div class="my-flex">
+                        <div class="flex-item">
+                            <img src="https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/03/ajwjl6u-540x270.jpg" alt="tnw-logo">
+                            <div class = "additional-content">
+                                <p>VW Wants its EVs to become part of the power grid when charging.</p>
+                                <small>Mathew Beedham <i class="fa fa-circle"></i>7 minutes ago</small>
+                            </div>
```

Comment:
- [ ] All the additional-content divs should have a link inside, not a `p` element. The text should be bold. Please change it in all the section's articles.

- [ ] The `small`  tag text should be gray and have a link in the author's name.